piker/data/_source.py

```diff
@@ -127,6 +134,56 @@ def unpack_fqsn(fqsn: str) -> tuple[str, str, str]:
     )
 
 
+class MktPair(Struct, frozen=True):
```

Comment:
Wrt this, i think if we want to have multiple endpoint brokers addressable eventually we can have type that maybe groups this type into some sort of address collection of sorts?